common/components/internal-header/_internal-header.scss

```diff
@@ -1,311 +1,182 @@
-$govuk-header-link-hover: govuk-colour("white");
+$govuk-header-text: govuk-colour('white');
+$govuk-header-link: govuk-colour('white');
+$govuk-header-link-hover: govuk-colour('white');
 $govuk-header-link-active: #1d8feb;
 $govuk-header-nav-item-border-color: #2e3133;
 
 .app-header {
-  @include govuk-font($size: 16);
-
-  color: $govuk-header-text;
-  background: $govuk-header-background;
-}
-
-.app-header__container--full-width {
-  padding: 0 govuk-spacing(3);
-  border-color: $govuk-header-border-color;
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+  background-color: $govuk-header-background;
 
-  .app-header__menu-button {
-    right: govuk-spacing(3);
+  @media print {
+    display: none;
   }
 }
 
-.app-header__container--with-border {
-  margin-bottom: -$govuk-header-border-width;
-  border-bottom: $govuk-header-border-width solid $govuk-header-border-color;
+.app-header__content {
+  flex-direction: row;
+  display: flex;
+  align-self: stretch;
 }
 
 .app-header__container {
-  @include govuk-clearfix;
-  position: relative;
-  padding-top: govuk-spacing(2);
-}
+  max-width: 1170px;
+  margin-right: govuk-spacing(3);
+  margin-left: govuk-spacing(3);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (min-width: 40.0625em) {
+    margin-right: govuk-spacing(6);
+    margin-left: govuk-spacing(6);
+  }
+
+  @supports (margin: Max(calc(0px))) {
+    margin-right: Max(
+      govuk-spacing(3),
+      calc(govuk-spacing(3) + env(safe-area-inset-right))
+    );
+    margin-left: Max(
+      govuk-spacing(3),
+      calc(govuk-spacing(3) + env(safe-area-inset-left))
+    );
+
+    @media (min-width: 40.0625em) {
+      margin-right: Max(
+        govuk-spacing(6),
+        calc(govuk-spacing(3) + env(safe-area-inset-right))
+      );
+      margin-left: Max(
+        govuk-spacing(6),
+        calc(govuk-spacing(3) + env(safe-area-inset-left))
+      );
+    }
+  }
 
-.app-header__logotype {
-  margin-right: govuk-spacing(1);
+  @media (min-width: 1230px) {
+    margin-right: auto;
+    margin-left: auto;
+  }
 }
 
 .app-header__logotype-crest {
-  margin-right: 1px;
-  fill: currentColor;
-  vertical-align: middle;
+  margin-right: govuk-spacing(2);
+  position: relative;
+  top: -2px;
+  fill: #fff;
 }
 
-.app-header__logotype-crest-fallback-image {
-  width: 42px;
-  height: 35px;
-  border: 0;
-  vertical-align: middle;
+.app-header__logo {
+  padding-right: govuk-spacing(3);
+  display: flex;
+  align-items: center;
 }
 
-.app-header__product-name {
-  @include govuk-font($size: 24);
-  display: inline-table;
+.app-header__logotype {
+  margin-right: govuk-spacing(2);
+  @include govuk-font($size: 24, $weight: bold);
+  display: flex;
+  align-items: center;
 }
 
 .app-header__link {
-  text-decoration: none;
+  @include govuk-font($size: 19);
 
   &:link,
-  &:visited {
-    color: $govuk-header-link;
+  &:visited,
+  &:active {
+    color: #fff;
```

Comment:
Do we need this change? The value of `$govuk-header-link` defined on line 5 is already white. I think it would be nice to continue to support the configuration approach.

Reply:
I knew I'd have missed one, I copied the styles from hmpps-auth and converted them to scss